crates/ty_python_semantic/src/util/get_size.rs

```diff
@@ -0,0 +1,13 @@
+use std::sync::Arc;
+
+use get_size2::GetSize;
+
+/// By default, `Arc<T>: GetSize` requires `T: 'static` to enable tracking references
+/// of the `Arc` and avoid double-counting. This method opts out of that behavior and
+/// removes the `'static` requirement.
+pub(crate) fn untracked_arc_size<T>(arc: &Arc<T>) -> usize
```

Comment:
The documentation doesn't make it entirely clear to me how an `Arc` is counted. From the code, my guess is that it counts the size of the arced value (it does double count if the arc is shared)